xrootdpyfs/fs.py

```diff
@@ -405,7 +407,7 @@ def removedir(self, path, recursive=False, force=False):
         status, _ = self._client.rmdir(self._p(path))
 
         if not status.ok:
-            directory_not_empty_error = status.errno == 3005
+            directory_not_empty_error = status.errno in [3005, 3018]
```

Comment:
it looks like that [3005](https://github.com/xrootd/xrootd/blob/5b5a1f6957def2816b77ec773c7e1bfb3f1cfc5b/src/XrdXrootd/XrdXrootdCallBack.cc#L198) is not that error, it is a bit misleading (it was there before).